lib/beaker/dsl/install_utils/module_utils.rb

```diff
@@ -133,8 +133,10 @@ def copy_module_to(one_or_more_hosts, opts = {})
               cur_path = File.join(target_module_dir, source_name)
               if (cur_path != target_path)
                 if host.is_powershell?
+                  on host, "rmdir /S /Q #{target_path}"
```

Comment:
Looking through the beaker host object, you can just do a beaker.rm_rm(target_path) and that will do the right thing per-OS.